INSTALL.md

```diff
@@ -0,0 +1,204 @@
+
+
+### Docker and Docker Compose
+
+So for now let's use Docker Compose. Be sure you have
+[Docker](https://docs.docker.com/engine/) and [Docker
+Compose](https://docs.docker.com/compose/) installed and the Docker daemon is
+running. Check if you have to run docker as local user or as root.
+
+    $ docker info
+
+
+### Get manage tool
+
+Go to a nice place in your filesystem and build or
+[download](https://github.com/OpenSlides/openslides-manage-service/releases) the
+manage tool `openslides`.
+
+You can also fetch it using `wget`. Don't forget to make the binary executable.
+
+    $ wget https://github.com/OpenSlides/openslides-manage-service/releases/download/latest/openslides
+    $ chmod +x openslides
+
+
+### Setup instance
+
+To setup the instance in this directory run:
+
+    $ ./openslides setup .
+
+This will give you a default setup with local self-signed SSL certificates. You
+will get [a browser warning and have to care yourself about checking the
+fingerprint of the
+certificate](https://en.wikipedia.org/wiki/Self-signed_certificate).
+
+[Below](#SSL-encryption) you find for more information and how to use
+[caddys](proxy) integrated certificate retrieval or how to disable the proxy
+expecting SSL. Disabling SSL is only feasible if you use an extra proxy in front
+of OpenSlides. Keep in mind that the browser client requires a HTTPS connection
+to the server. It is NOT possible to use OpenSlides without any SSL encryption
+at all.
+
+Now have a look at the `docker-compose.yml`. You can customize it if you want
+but for most cases this is not recommended.
+
+
+### Pull images and start services
+
+Then run:
+
+    $ docker-compose pull
+    $ docker-compose up
+
+
+### Initialize database
+
+Wait until all services are available. Then run in a second terminal in the same
+directory:
+
+    $ ./openslides initial-data
+
+Now open https://localhost:8000, login with superuser credentials (default
+username and password: `superadmin`) and have fun.
+
+
+### Reach OpenSlides from outside your local machin
+
+Normally you want OpenSlides be reachable from outside your local machine. You
+have several options to achieve this:
+
+1. Use a [custom YAML configuration
+   file](#Configuration-of-the-generated-Docker-Compose-YAML-file) to change
+   `host` to a public device (e. g. `0.0.0.0`). If you also want to change the
+   `port`, keep in mind that port numbers less than 1024 normally need root
+   privileges on Unix systems.
+
+2. Use a proxy server like Apache HTTP Server or Nginx. In this case your proxy
+   should handle SSL-encryption and you should disable OpenSlides local
+   SSL-encryption (see [respective section below](#SSL-encryption)).
+
+
+
+## Stop the server and remove the containers
+
+To stop the server run:
+
+    $ docker-compose stop
+
+To remove all containers run:
+
+    $ docker-compose rm
+
+To remove the database you have to remove the content of the `db-data`
+directory.
+
```

Comment:
To do a deeper cleanup (networks and containers) there is 
    $ docker-compose down

I think we should add this here.